recipes/clickhouse-cpp/all/conanfile.py

```diff
@@ -0,0 +1,129 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain,CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.build import check_min_cppstd
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.51.3"
```

Comment:
```suggestion
required_conan_version = ">=1.53.0"
```

options.rm_safe was added on 1.53.0: https://docs.conan.io/1/changelog.html#oct-2022